skrf/tests/test_network.py

```diff
@@ -441,6 +441,14 @@ def test_cascade2(self):
         self.assertEqual(self.ntwk1 >> self.ntwk2, self.ntwk3)
         self.assertEqual(self.Fix2 >> self.DUT2 >> self.Fix2.flipped(), self.Meas2)
 
+    def test_concat_ports(self):
+        l2 = self.cpw.line(0.07, 'm', z0=50)
```

Comment:
Store `l2` in class (`self`) rather than duplicating it here.

Reply:
Changes made as requested.